src/components/NavLandData.js

```diff
@@ -41,11 +41,16 @@ const NavLandData = ({ open, active, onClose }) => {
     const activeGroups = useSelector((state) => state.dataGroups.activeGroups);
     const displayProperties = useSelector((state) => state.landOwnership.displayActive);
 
+    const description = <p className='land-data-description'>
+        Want to add your own data to Land Explorer? <a href="https://landexplorer.coop/#contact">Contact Us.</a>
```

Comment:
Haven't tested whether it opens in a new tab, but if not, should we add `target="_blank"` so that current changes aren't lost?

Reply:
Good idea, added that